eln-check.py

```diff
@@ -148,5 +149,33 @@ def diff_with_rawhide(package, eln_build=None, rawhide_build=None):
 
     f.close()
     s.close()
+    os.system("sort -u -o %s %s" % (args.status, args.status))
 
     logging.info("Total differences {0}".format(counter))
+
+    # Create Webpage
```

Comment:
IMHO better to use jinja template for this

Reply:
OK, I'll look at that.